Common/CodeBlock.h

```diff
@@ -10,6 +10,14 @@
 #include "Common/Log.h"
 #include "Common/MemoryUtil.h"
 
+#if PPSSPP_PLATFORM(SWITCH)
+#include <cstdio>
+#include <switch.h>
+// This is a bit of a whacky solution, however this is the simplest implementation for it
+// given the restrictions in Horizon
+extern Jit* activeJitController;
```

Comment:
This isn't needed anymore right?  I haven't looked at the rest much, just want to make sure this is an artifact from rebasing still.  Doesn't look like it's actually used anymore, aside from setting it.

-[Unknown]

Reply:
Sorry for late answer, yea i think this is a leftover from when writable/exec memory werent properly split in the emitter. Will clean that up ext time

Reply:
Removed in next rebase